ingenialink/canopen/register.py

```diff
@@ -33,23 +35,25 @@ class CanopenRegister(Register):
 
     def __init__(
         self,
-        idx,
-        subidx,
-        dtype,
-        access,
-        identifier=None,
-        units=None,
-        cyclic="CONFIG",
-        phy=REG_PHY.NONE,
-        subnode=1,
-        storage=None,
-        reg_range=(None, None),
-        labels=None,
-        enums=None,
-        cat_id=None,
-        scat_id=None,
-        internal_use=0,
-        address_type=None,
+        idx: int,
+        subidx: int,
+        dtype: REG_DTYPE,
+        access: REG_ACCESS,
+        identifier: Optional[str],
```

Comment:
Default values are missing

Reply:
Which default values are missing?

Reply:
It was fixed in this [commit ](https://github.com/ingeniamc/ingenialink-python/pull/319/commits/a89c27f89a5185876cea5e4768d07c9d7a66bfa0)